src/Layout.tsx

```diff
@@ -251,56 +277,176 @@ const styles =
               display: none;
             }
+          }
+          .navbar-center {
+            flex: none;
+            width: ${theme.spacing(47)};
+
+            .center-container {
+              width: ${theme.spacing(19.5)};
+              margin: auto;
+              a {
+                display: block;
+              }
+            }
 
-            a {
-              margin-left: ${theme.spacing(8)};
-              color: ${theme.palette.white.main};
-              text-decoration: none;
+            ${theme.breakpoints.down("mobile")} {
+              width: auto;
+              flex: 1 1 0;
+
+              .center-container {
+                margin: ${theme.spacing(0, 0, 0, 3)};
+              }
             }
           }
+          .navbar-right {
+            flex: 1 1 0;
 
-          .mobile-menu-button {
-            padding: 0;
-            background: none;
-            border: none;
+            ${theme.breakpoints.down("mobile")} {
+              flex: 0;
+              margin-right: ${theme.spacing(3)};
+            }
 
-            ${theme.breakpoints.up("mobile")} {
-              display: none;
+            .header-links {
+              flex-grow: 1;
+              text-align: end;
+              display: flex;
+              align-items: center;
+
+              .link-spacer {
+                flex: 1 1 0;
+              }
+
+              ${theme.breakpoints.down("mobile")} {
+                display: none;
+              }
+
+              a {
+                margin-left: ${theme.spacing(8)};
+                color: ${theme.palette.white.main};
+                height: 100%;
+                text-decoration: none;
+                text-transform: uppercase;
+                letter-spacing: 0.04em;
+                font-size: ${theme.typography.operatorBrowserNameHeading
+                  .fontSize}px;
+                line-height: ${theme.typography.operatorBrowserNameHeading
+                  .lineHeight};
+                font-weight: ${theme.typography.operatorBrowserNameHeading
+                  .fontWeight};
+              }
             }
-          }
 
-          .mobile-menu {
-            align-self: center;
-            position: relative;
-            top: -2px;
+            .mobile-menu-button {
+              padding: 0;
+              background: none;
+              border: none;
+
+              ${theme.breakpoints.up("mobile")} {
+                display: none;
+              }
+            }
+
+            .mobile-menu {
+              align-self: center;
+              position: relative;
+              top: -2px;
+            }
           }
         }
+      }
+
+      header {
+        padding: ${theme.spacing(3, 3, 0)};
+
+        ${theme.breakpoints.down("mobile")} {
+          padding: ${theme.spacing(2, 2, 0)};
+        }
 
         .heading-and-breadcrumb {
           display: flex;
-          flex-direction: column-reverse;
+          flex-direction: column;
```

Comment:
I want to say I did `column-reverse` here on purpose so as to present the raw HTML (e.g. to a screen reader, or web crawler, etc.) in a more logical fashion–that is, the `<h1>` before the breadcrumb. I don't actually know if that's an improvement or not, but I thought I'd comment why.

Reply:
Hrm. I'm not actually sure either.

Honestly, they don't seem that different to me. I might just use `column` because it's easier for the HTML itself.